src/libponyrt/actor/actor.c

```diff
@@ -757,8 +878,10 @@ PONY_API void pony_become(pony_ctx_t* ctx, pony_actor_t* actor)
 
 PONY_API void pony_poll(pony_ctx_t* ctx)
```

Comment:
I think that this function isn't called anywhere.  If so, then it can by removed and also the `polling` arg to `ponyint_actor_run` can be removed

Reply:
My understanding is that this is for when Pony is used as a C library.

Reply:
Correct